riptide-spring-boot-autoconfigure/src/test/java/org/zalando/riptide/autoconfigure/OpenTelemetryTestAutoConfiguration.java

```diff
@@ -0,0 +1,14 @@
+package org.zalando.riptide.autoconfigure;
+
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.api.trace.TracerProvider;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class OpenTelemetryTestAutoConfiguration {
```

Comment:
Do we still need this class after the change to the OpenTelemetryPlugin constructor?